featuretools/feature_base/feature_base.py

```diff
@@ -328,21 +328,23 @@ def unique_name(self):
 class IdentityFeature(FeatureBase):
     """Feature for entity that is equivalent to underlying variable"""
 
-    def __init__(self, variable):
+    def __init__(self, variable, name=None):
```

Comment:
This is code from PR #580 
We should make sure that the PR stands by itself, and doesn't contain commits from other PRs